### PR TITLE
use 67k inputdata by default

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**
## :bird: Description of this PR :bird:

- This PR sets the 67k inputdata rev4.94 as default
- No calibration dataset is set in cfg$input['calibration'] so this PR should only be merged after #609 
- The USA forest dynamic we observed in the magpie meeting is no longer present thanks to a bugfix in the npi ndc script

## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

- [x] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [x] Document changes 
  - Add changes to `CHANGELOG.md`
  - Where relevant, put In-code documentation comments
  - Properly address updates in interfaces in the module documentations
  - run [`goxygen::goxygen()`](https://github.com/pik-piam/goxygen) and verify the modified code is properly documented

- [x] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `Rscript start.R --> "compilation check"`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
  - **High risk**:
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
    - Default run from the PR target branch for comparison
    - Provide relevant comparison plots (land-use, emissions, food prices, land-use intensity,...)

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : ** mins
  - This PR's default :  ** mins

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate
- [ ] content review done (at least 1)
- [x] RSE review done (at least 1)

### CODE REVIEW

**Review by tscheypidi — APPROVED** [review]
Looks good

**Comment by FelicitasBeier** [comment]
> * The USA forest dynamic we observed in the magpie meeting is no longer present thanks to a bugfix in the npi ndc script

Which script was that? In the preprocessing? Because I cannot see the change here in the GAMS code.

**Comment by pfuehrlich-pik** [comment]
The npi ndc fix happened here https://github.com/magpiemodel/magpie/pull/608

File: config/default.cfg [hunk 1]
```diff
@@ -22,11 +22,10 @@ cfg$model <- "main.gms"   #def = "main.gms"
...
 # which input data sets should be used?
-cfg$input <- c(regional    = "rev4.93_h12_magpie.tgz",
-               cellular    = "rev4.93_h12_fd712c0b_cellularmagpie_c200_MRI-ESM2-0-ssp370_lpjml-8e6c5eb1.tgz",
-               validation  = "rev4.93_h12_validation.tgz",
-               additional  = "additional_data_rev4.46.tgz",
-               calibration = "calibration_H12_per_ton_fao_may22_glo_08Aug23.tgz")
+cfg$input <- c(regional    = "rev4.94_h12_magpie.tgz",
+               cellular    = "rev4.94_h12_fd712c0b_cellularmagpie_c200_MRI-ESM2-0-ssp370_lpjml-8e6c5eb1.tgz",
+               validation  = "rev4.94_h12_validation.tgz",
+               additional  = "additional_data_rev4.46.tgz")
```
[the remark this answers is not in the document]
Comment (tscheypidi) [reply config/default.cfg:28]: good point. I checked the file and I think you are right....land cost calibration information also seems to be part of the calibration file

[the remark this answers is not in the document]
Comment (FelicitasBeier) [reply config/default.cfg]: Alright. I reintroduced the old calibration.tgz now.

**Review by FelicitasBeier — APPROVED** [review]
Good to go.

**Pull request merged**